rfc/2022/2022-06-15 - Keycloak configuration managed by Terraform.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Keycloak configuration will be managed by Terraform to support automation 
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2022-06-15`
+* Team Crew and Name: Platform/Access Management Team
+* Authors:
+  * [Ritesh Patel](https://github.com/iriteshp)
+  * [Kevin Duensing](https://github.com/kjduensing)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/14)
+
+## Background
+The Infrastructure as Code (IaC) is managing and provisioning the infrastructure as a code instead of doing it manually. Configurations are created as a code containing specifications of resources, which makes it easier to create, update, manage and distribute the configurations. There are several IaC solutions available currently but we will implement Terraform as IaC to configure Keycloak resources automatically because it is cloud-agnostic and provides a flexible abstraction of resources and providers
```

Comment:
```suggestion
Infrastructure as Code (IaC) is the practice of managing and provisioning infrastructure programmatically instead of doing it manually. Configurations are created as code containing resource specifications, making them easier to create, update, manage and distribute. Several IaC solutions are currently available, but we will implement Terraform as IaC to configure Keycloak resources automatically because it is cloud-agnostic and provides a flexible abstraction of resources and providers.
```